tests/functional/test_mandel.py

```diff
@@ -61,17 +61,17 @@ def results() -> list[MandelSaveData]:
     # t = 25 [s]
     DesiredError(
         error_pressure=0.02468899282842615,
-        error_flux=0.4277241964548815,
+        error_flux=0.362852,
         error_displacement=0.0007426275934204356,
-        error_force=0.007199561782640474,
+        error_force=0.006999,
```

Comment:
Interesting that most of the values do not differ too much after the change. I would have not expected that. But the math looks correct!

Reply:
Thanks for a thorough check! I believe the explanation is that we use relative errors with relatively homogeneous values, so the measures cancel between numerator and denominator.